api/tests/integration/infrastructure/repositories/user-repository_test.js

```diff
@@ -443,6 +443,15 @@ describe('Integration | Infrastructure | Repository | UserRepository', () => {
       // then
       expect(result).to.be.instanceOf(AlreadyRegisteredEmailError);
     });
+
+    it('should reject an AlreadyRegisteredEmailError when email case insensitive already exists', async () => {
+      // when
+      const capitalEmailAlreadyInDb = userInDb.email.toUpperCase();
```

Comment:
Cette ligne devrait être dans une section given ?

Reply:
D'ailleurs autour de ce sujet : je mets mes `// given` dans le context, mais pas mes `// when`, peut-être à tort.
Est-ce qu'on ne devrait pas garder uniquement le `// then` dans nos blocs it ?